src/test/junitParser.test.ts

```diff
@@ -0,0 +1,51 @@
+import assert from 'node:assert/strict';
+import { describe, it } from 'node:test';
+import { parseTestReports } from '../junitParser';
+
+const DIRNAME = __dirname;
+const getFirstSuite = (fileName: string) => ({
+    summary: 'test',
+    checkName: 'first suite',
+    fileName: `${DIRNAME}/${fileName}`,
+    totalCount: 2,
+    skipped: 1,
+    failedEvaluating: 1,
+    annotations: [
+        {
+            testStatus: 'draft',
+            isTestimTest: true,
+            annotation_level: 'notice',
+            title: 'first suite first test',
+            message: 'https://app.testim.io/#/project/coco/branch/master/test/1234567A?result-id=1234567A',
+            raw_details: 'test - first suite first test:\n(undefined)',
+            path: 'https://app.testim.io/#/project/coco/branch/master/test/1234567A?result-id=1234567A',
+            end_line: 1,
+            start_line: 1,
+        },
+        {
+            testStatus: 'draft',
+            isTestimTest: true,
+            annotation_level: 'failure',
+            title: 'first suite second test',
+            message: 'Step Failed: Element is not visible More info at: \nhttps://app.testim.io/#/project/coco/branch/master/test/1234567B?result-id=1234567B',
+            raw_details: 'test - first suite second test:\n(Step Failed: Element is not visible More info at: https://app.testim.io/#/project/coco/branch/master/test/1234567B?result-id=1234567B)',
+            path: 'https://app.testim.io/#/project/coco/branch/master/test/1234567B?result-id=1234567B',
+            end_line: 1,
+            start_line: 1,
+        },
+    ],
+    failed: 1,
+    passed: 0,
+});
```

Comment:
Could we simplify this with the test-runner snapshots?
See https://nodejs.org/docs/latest/api/test.html#snapshot-testing

Reply:
Do you have a project that uses this?
I couldn't find how to use it and the docs example doesn't work for me